src/EnergyPlus/WaterCoils.cc

```diff
@@ -2143,6 +2143,23 @@ namespace WaterCoils {
                     ShowContinueError("The autosize value for max air volume flow rate is zero");
                 }
 
```

Comment:
Needed to move the humrat call to before the temp call.

Reply:
Well I can't see now why this had to be moved. At the time I recall an uninitialized variable but reviewing again I can't see what that might be. It doesn't hurt anything to size inlet air hum rat before inlet air temp since they are each independent of each other.

Reply:
It doesn't hurt anything other than it causes reordering in some of the output.

Reply:
It was necessary because CoolingWaterDesAirInletTempSizing now needs to know the humrat to calculate fan heat delta T.

    } else if (SizingType == CoolingWaterDesAirInletTempSizing) {
                // calculate fan heat to get fan air-side delta T
                FanCoolLoad = DataAirSystems::calcFanDesignHeatGain(DataFanEnumType, DataFanIndex, DataAirFlowUsedForSizing);
                if (DataDesInletAirHumRat > 0.0 && DataAirFlowUsedForSizing > 0.0) {
                    CpAir = PsyCpAirFnWTdb(DataDesInletAirHumRat, AutosizeDes);
                    fanDeltaT = FanCoolLoad / (CpAir * StdRhoAir * DataAirFlowUsedForSizing);
                }